### DISCUSSION TITLE
A GUI front end example....

### DISCUSSION THREAD

**Discussion by MikeTheWatchGuy** [mido/mido#416] [Show and tell]
I have created a GUI front-end for `mido` using a tkinter wrapper packaged named [PySimpleGUI](https://pysimplegui.readthedocs.io/en/latest/).

If there any interest in posting it in an examples section or referencing it in a document?

Here are a couple of screenshots.  There are 2 screens to the GUI... one gets the file or folder name.  The other is the main player interface.

If there is a more appropriate way to post about this, please let me know.  At the moment, the demo file is simply part of the Master Branch in the [GitHub](https://github.com/MikeTheWatchGuy/PySimpleGUI).

![midi player get files](https://user-images.githubusercontent.com/13696193/44583920-3bbc2b80-a775-11e8-99c8-d9fcc6d31b9d.jpg)

![midi player playback](https://user-images.githubusercontent.com/13696193/44583921-3c54c200-a775-11e8-9d82-9624fe103c29.jpg)

If it's not appropriate for me to post this here, then by all means delete the issue.

**Comment by PySimpleGUI** [comment]
I just made a Repo for the program I use to playback MIDI files now.  It's not the best Python code as it was written when I had just started to learn Python.

https://github.com/MikeTheWatchGuy/MidiLooper

**Reply by MikeTheWatchGuy** [reply]
This is how the interface looks now.

![MidiLooper](https://user-images.githubusercontent.com/13696193/213475459-e095c158-0884-4ba4-9d51-015b579bceba.png)
